assets/js/phoenix_live_view/view.js

```diff
@@ -1092,7 +1092,10 @@ export default class View {
         .filter(form => form.elements.length > 0)
         .filter(form => form.getAttribute(this.binding(PHX_AUTO_RECOVER)) !== "ignore")
         .map(form => {
-          let newForm = template.content.querySelector(`form[id="${form.id}"][${phxChange}="${form.getAttribute(phxChange)}"]`)
+          // attribute given via JS module needs to be escaped as it contains the symbols []",
+          // which result in an invalid css selector otherwise.
+          const phxChangeValue = form.getAttribute(phxChange).replaceAll(/([\[\]"])/g, '\\$1')
```

Comment:
Again, I could use `CSS.escape` here instead. 

https://developer.mozilla.org/en-US/docs/Web/API/CSS/escape

But, it's not supported in the node env, so we would have to stub and spy on it, and not actually have a full input-output test.